src/controller/src/clusters.rs

```diff
@@ -449,7 +450,16 @@ where
                             format!("--internal-http-listen-addr={}", assigned["internal-http"]),
                             format!("--opentelemetry-resource=cluster_id={}", cluster_id),
                             format!("--opentelemetry-resource=replica_id={}", replica_id),
-                        ]
+                        ];
+
+                        if let Some(instance_directory) = &instance_directory {
+                            args.push(format!(
+                                "--instance-directory={}",
+                                instance_directory.display()
+                            ));
+                        }
```

Comment:
This doesn't seem like it will work correctly for the process orchestrator—they'll all end up sharing the same instance directory, no?

Reply:
Within the instance directory, separate subdirectories are being created for each source -- `u1234`, `u2345` and so on. So it seems that multiple computeds can successfully coexist inside the same container